packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -36,82 +49,132 @@ const useStyles = makeStyles({
   filledInteractive: {
     cursor: 'pointer',
     backgroundColor: tokens.colorNeutralBackground1,
-    ...shorthands.borderColor(tokens.colorTransparentStroke),
     boxShadow: tokens.shadow4,
 
+    '::after': {
+      ...shorthands.borderColor(tokens.colorTransparentStroke),
+    },
+
     ':hover': {
```

Comment:
out of topic for this PR, but what about `:focused`  ? also all hover states should be probably turned off on touch devices, do we have any guidance on that from design ?

Reply:
Focus indicators have yet to be tackled. The goal is to use indicators as all other components do.
Regarding touch devices, I haven't yet read anything explicitly about it.

Reply:
k, ty